bin/node-template/pallets/template/Cargo.toml

```diff
@@ -27,7 +27,9 @@ sp-io = { version = "23.0.0", path = "../../../../primitives/io" }
 sp-runtime = { version = "24.0.0", path = "../../../../primitives/runtime" }
 
 [features]
-default = ["std"]
+default = [
+	"std",
```

Comment:
I really dislike this formatting style. In pure rust, when 1 line is sufficient to hold all elements in a list, then it uses a one-liner. Here, it's unconditionally forcing all lists to be separated to at least 3 different lines.

Reply:
I don't mind it personally. it means when we go from 1 to >1, or >1 to 1 elements in the list, we don't need to adjust the formatting.

Reply:
I also don't mind much. 
I mean one liner looks a bit nicer. But on the other hand, on a three liner when an element is added the diff will be nicer to read.

Reply:
I changed the format now to use normal 80 line width, since in 99% of cases the `default` really just contains `std`.